README.md

```diff
@@ -69,14 +68,14 @@ Run the tool with the -h flag to see a list of available commands:
 	{create_bdev,delete_bdev,create_subsystem,delete_subsystem,create_namespace,delete_namespace,add_host,delete_host,create_listener,delete_listener,get_subsystems}
 
 	optional arguments:
-	-h, --help            			show this help message and exit
-	-c CONFIG, --config CONFIG
-			      			Path to config file
+    -h, --help            show this help message and exit
+    -s SETTINGS_FILE, --settings_file SETTINGS_FILE
+                          Path to settings file
```

Comment:
`--settings_file` looks weird to me. I understand you renamed config to settings to avoid confusion with the persistent config, but I don't remember I have seen an application that has an option with such a name. Usually it is `--config`. If we really want to avoid confusion with the persistent config I would rather rename the persistent config to something else. Like `nvme.db`?

Reply:
@trociny I'm not sure I quite understand what you mean by `nvme.db` it's not necessarily about the file name it is about avoiding confusion in the code in respect to variable names and class names. Alternatively we can try to find another name for the persistent config.

Reply:
I am talking about another name for the persistent config. I.e. `DB` instead of `PersistenConfig`, and `nvme.db` object name instead of  `nvme.config`.

Reply:
Got it. `DB` is a bit too generic for my taste. Maybe something like `GWState` or `GroupState`? Any other ideas?

Reply:
For the internal class names we can pick literally any name.  For the RADOS object name, in the iSCSI gateway it is actually configurable (in what is here referred to as a "settings file").

Reply:
The same here, the group file in the settings file (implicitly) defines the RADOS object name:
`self.omap_name = f"nvme.{gateway_group}.config"`

Reply:
The reason I initially asked to change from config to something else was that the code was confusing because there were two config types floating around and it was not always obvious which was which. So the idea was to give one a different name so we can clearly differentiate.

Reply:
> The same here, the group file in the settings file (implicitly) defines the RADOS object name: `self.omap_name = f"nvme.{gateway_group}.config"`

Not quite, in the iSCSI gateway the entirety of the object name is up to the user -- they could make it "foobar".  But then, the iSCSI gateway doesn't have a concept of a gateway group so it doesn't need any sort of structure in the name.

BTW is the character set for the gateway group name is constrained?  If not, it should be, to avoid weird object names if nothing else.

> The reason I initially asked to change from config to something else was that the code was confusing because there were two config types floating around and it was not always obvious which was which. So the idea was to give one a different name so we can clearly differentiate.

Makes sense but I agree with Mykola.  The plaintext config file and related CLI options should continue to be termed config(uration), have `.conf` or `.config` extension, etc.  Let's rename the persistent "config" bits instead.

Reply:
> Got it. `DB` is a bit too generic for my taste. Maybe something like `GWState` or `GroupState`? Any other ideas?

`DB` was just an example. I am fine with any other name.